source/includes/api-reference/_api-reference.html.md

```diff
@@ -131,7 +131,103 @@ GET /v1/{userName}/{brainName}
 | user | Name of the user who has the BRAIN|
 | name | Name of the BRAIN |
 
+## BRAIN Status Websocket
 
+Use a websocket connection to get live updates about a BRAIN.
+
+> Request
+
+```text
+GET /v1/{userName}/{brainName}/ws'
+```
+
+| Parameter | Description |
+| --- | --- |
+| userName | Name of the user who has the BRAIN |
+| brainName | Name of the BRAIN |
+
+> Example JSON Response
+
+```json
+{
+    "type": "ADD_DATA_POINT",
+    "metric": "episode_value",
+    "value": 34.0,
+    "episode": 83,
+    "concept": "balance",
+    "lesson": "balancing"
+}
+
+{
+    "type": "PROPERTY_CHANGED",
+    "property": "status",
+    "value": "In Progress"
+}
+
+{
+    "type": "CONCEPTS_SET",
+    "concepts": [{
+        "name": "balance",
+        "state": "Not Started",
+    }],
+}
+
+{
+    "type": "CONCEPT_CHANGED",
+    "concept": "balance",
+    "state": "In Progress"
+}
+
+{
+    "type": "FILE_UPDATED",
+    "filename": "cartpole.ink",
+    "hash": "c06edf7bc96dd461af1357a274734633b0ff2932"
+}
+```
+
+### Response
+
+The websocket will send JSON messages for events in the BRAIN. Every message will have a `type` field which can be used to determine what the rest of the payload is.
+
+There are 6 types of messages that are sent on this socket: `ADD_DATA_POINT`, `PROPERTY_CHANGED`, `CONCEPTS_SET`, `CONCEPT_CHANGED`, `FILE_UPDATED`, `FILES_UPDATED`, and `TRAINING_INITIALIZED`.
+
+## ADD_DATA_POINT
```

Comment:
These can't be second level headings if they are nested under a second level heading. Is the idea for them to be nested under the `### Response` heading? If so they should be a #### heading.